spec/requests/api/v1/teachers/courses/prizes/prize_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe 'Teacher Course Prize' do
+  it 'creates a new prize' do
```

Comment:
I see all three of your tests here. They look great! I pulled down and tested as well, all working well!